pyconcz_2018/proposals/models.py

```diff
@@ -71,7 +71,7 @@ class Talk(EntryBase):
 
     LANGUAGES = (
         ('en', 'English (preferred)'),
-        ('cs', 'Czechoslovak'),
+        ('cs', 'Czech/Slovak'),
```

Comment:
FWIW, `cs` je mezinarodni kod pro cestinu

Reply:
hm, to je fakt, ale tak to je interní pojmenovaní, ne? nebo to chceš změnit na csk? :D